classes/endpoint.php

```diff
@@ -51,7 +53,7 @@ public function get_oembed_response( WP_REST_Request $request ) {
 			return new WP_Error( 'rest_oembed_invalid_url', __( 'Invalid URL.', 'oembed-api' ), array( 'status' => 404 ) );
 		}
 
-		if ( 'json' !== $request['format'] ) {
+		if ( ! in_array( $request['format'], array( 'json', 'xml' ) ) ) {
 			return new WP_Error( 'rest_oembed_invalid_format', __( 'Invalid format.', 'oembed-api' ), array( 'status' => 501 ) );
 		}
```

Comment:
Arg validation should be able to return a `WP_Error` (in addition to the standard true/false) that gets used by the API as the error. If not, file an issue. :)